app/components/provider_interface/course_summary_component.rb

```diff
@@ -0,0 +1,35 @@
+module ProviderInterface
+  class CourseSummaryComponent < ViewComponent::Base
+    attr_reader :application_choice, :provider_name, :course_name_and_code,
+                :location_name_and_address, :study_mode
+
+    def initialize(application_choice:)
+      @application_choice = application_choice
+      @provider_name = application_choice.provider.name
+      @course_name_and_code = application_choice.course.name_and_code
+      @location_name_and_address = application_choice.course_option.site.name_and_address
+      @study_mode = application_choice.course_option.study_mode.humanize
+    end
+
+    def course_summary_rows
```

Comment:
Consider renaming this to rows. As the component name indicates its purpose I think we can simplify this method name.